### PR TITLE
Fixed Issue #4 - Added Splash Screen UI

### PR DESCRIPTION
**Pull request by AminullahTajMuhammad**
## Description
Implemented Splash Screen. Here is the Screenshot

Splash Screen UI

## Type of Changes?
Just put an x in the [ ] which are valid.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing Suites

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
- [x] ./gradlew assembleDebug assembleRelease

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -3,4 +3,6 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
+
```
Comment (iadeelzafar) [inline app/src/main/res/values/colors.xml]: Remove this blank line.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">ChatAir-android</string>
+    <string name="txt_chat_air">ChatAir</string>
```
Comment (iadeelzafar) [inline app/src/main/res/values/strings.xml]: **text_chatair**

**Comment by AminullahTajMuhammad** [comment]
@iadeelzafar Fixed Changes. Here is a Screenshot.
Splash Screen UI Updated

**Comment by iadeelzafar** [comment]
The font looks very small. Please look into its size.
Other than that everything else looks good .

**Comment by iadeelzafar** [comment]
Also, have you used Square to reformat the changed files?

**Comment by AminullahTajMuhammad** [comment]
@iadeelzafar As per our initial discussions, Square or any kind of code style is not supposed to be integrated at this time. We will be adding these when we integrate Travis CI at some later point.

**Comment by iadeelzafar** [comment]
@AminullahTajMuhammad gotcha! let me remove the codestyle guidelines from readme then.

**Comment by AminullahTajMuhammad** [comment]
@iadeelzafar Anything else to do?

**Pull request merged**